packages/firebase_storage/firebase_storage_web/lib/src/interop/storage_interop.dart

```diff
@@ -8,225 +8,264 @@
 @JS('firebase_storage')
 library firebase.storage_interop;
 
+import 'dart:js_interop';
+
 import 'package:firebase_core_web/firebase_core_web_interop.dart';
-import 'package:js/js.dart';
 
 @JS()
-external StorageJsImpl getStorage([AppJsImpl? app, String? bucketUrl]);
+@staticInterop
+external StorageJsImpl getStorage([AppJsImpl? app, JSString? bucketUrl]);
 
 @JS()
+@staticInterop
 external void connectStorageEmulator(
-    StorageJsImpl storage, String host, int port,
+    StorageJsImpl storage, JSString host, JSNumber port,
     [EmulatorOptions? options]);
 
 @JS()
-external PromiseJsImpl<void> deleteObject(ReferenceJsImpl ref);
+@staticInterop
+external JSPromise deleteObject(ReferenceJsImpl ref);
 
 @JS()
-external PromiseJsImpl<String> getDownloadURL(ReferenceJsImpl ref);
+@staticInterop
+external JSPromise getDownloadURL(ReferenceJsImpl ref);
 
 @JS()
-external PromiseJsImpl<String> getBlob(ReferenceJsImpl ref,
-    [int? maxDownloadSizeBytes]);
+@staticInterop
```

Comment:
Noticed we aren't putting the return type in a code comment anymore. Might be useful for our future selves?

Reply:
I've added typing in comments :)